go/vt/wrangler/schema.go

```diff
@@ -333,7 +333,7 @@ func (wr *Wrangler) CopySchemaShardFromShard(ctx context.Context, tables, exclud
 		return fmt.Errorf("GetShard(%v, %v) failed: %v", sourceKeyspace, sourceShard, err)
 	}
 	if sourceShardInfo.MasterAlias == nil {
-		return fmt.Errorf("no master in shard record %v/%v. Consider running 'vtctl InitShardMaster' in case of a new shard or to reparent the shard to fix the topology data, or providing a non-master tablet alias", sourceKeyspace, sourceShard)
+		return fmt.Errorf("no master in shard record %v/%v. Consider running 'vtctl InitShardPrimary' in case of a new shard or to reparent the shard to fix the topology data, or providing a non-master tablet alias", sourceKeyspace, sourceShard)
```

Comment:
should we change the other instances of "master" here (and the error below) to "primary"?

Reply:
Good catch, I'll fix them.